css/style.css

```diff
@@ -5,15 +5,195 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+/*Header
+-----------------------------------------------------*/
+.header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 10px 50Px ;
```

Comment:
Sizing should be specified in relative units (`em`/`rem` rather than `px`)

Reply:
Done!